_posts/2020-09-02-solidity-0.7.1-release-announcement.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.1 Release Announcement'
+date: '2020-09-02'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.7.1](https://github.com/ethereum/solidity/releases/tag/v0.7.1) .
+
+
+## Notable New Features 
+
+### Functions At File-Level
+
+Functions can now be defined at file-level. Such functions are also called "free functions"
+(as opposed to functions bound to a specific contract).
+
+These functions will always be internal functions
+and are meant to replace internal library functions and their very special behaviour.
+
+These functions can do everything an internal library function can do, which includes
+calling other file-level functions, emitting events, sending Ether,
+calling functions on other contarcts "in the name" of the current contract
```

Comment:
```suggestion
calling functions on other contracts "in the name" of the current contract
```

Reply:
Also I'm not sure I like ``"in the name" of the current contract``...
Maybe ``"in the name of the current contract" (i.e. as if the function was an internal function of the current contract)``?

Actually, maybe ``Similarly to internal library functions, free functions behave as if they were internal functions of the contract calling them, i.e. they can ... .`` or something like that might simplify the whole thing...